src/literals/infinity.es6

```diff
@@ -0,0 +1,24 @@
+import CheddarPrimitive from './primitive';
+import * as CheddarError from '../consts/err';
+
+import {ClassType} from '../consts/types';
+
+export default class CheddarInfinityToken extends CheddarPrimitive {
+    exec() {
+        if (this.curchar === 'I' &&
+            this.Code[this.Index + 1] === 'n' &&
+            this.Code[this.Index + 1] === 'f' &&
```

Comment:
As @ETHproductions said, i don't think these should all be the same. You can also use `this.jumpLiteral("Infinity")` which returns false if not a match, truthy otherwise